src/routes/TasteScoresChartByApple.svelte

```diff
@@ -80,32 +80,62 @@
   });
 
   let tooltipParentEl;
+
+  let annotationText;
+  $: {
+    if ($currentRankFilter === 'better') {
+      annotationText =
+        'Not only was <strong>Wild Twist</strong> our favorite apple, but it also showed the biggest improvement between expected and actual taste.';
+    }
+    if ($currentRankFilter === 'same') {
+      annotationText =
+        'The <strong>Evercrisp</strong> might rank as the most boring apple. We expected it to taste average...and it did.';
+    }
+    if ($currentRankFilter === 'worse') {
+      annotationText =
+        '<strong>Opal Gold</strong> was our biggest disappointment. It lost nearly a whole point after we tasted it.';
+    }
+  }
 </script>
 
 <div class="header">
-  <div class="title-with-sort">
-    <h2>Apples that tasted</h2>
-    <select on:change={e => currentRankFilter.set(e.target.value)} value={$currentRankFilter}>
-      {#each ranks as rankObject}
-        <option value={rankObject.key}>{rankObject.label}</option>
-      {/each}
-    </select>
+  <p>
+    As the old saying goes, you eat first with your eyes. Before taking a bite, we judged the apples
+    by the way we <em>expected</em> them to taste. We compare that score to the actual taste score below.
+  </p>
```

Comment:
Not important but I actually liked it better how you had it before (on the pictures you sent to the Slack group), showing this between the header and the charts, I also feel that this follow the style of the rest of the page

Reply:
Good suggestion! Will make that change